webpack.config.js

```diff
@@ -61,12 +53,12 @@ module.exports = {
   },
   devtool: '#eval-source-map',
   plugins: [
-    // provide values to application from process.env
-    new webpack.DefinePlugin({
-      'process.env': Object.assign({ 
-        NODE_ENV: JSON.stringify(process.env.NODE_ENV)
-      }, env)
-    }),
+    // provide values to application from process.env, only providing
+    // values referenced in codebase, which is safer than providing all env
+    // values
```

Comment:
is this line a typo?

Reply:
nope! that's how I meant it

Reply:
oh okay it just looked weird because this comment block is wrapping differently than the one up by line 19. 🤷‍♀️